climpred/metrics.py

```diff
@@ -32,7 +34,7 @@ def _get_norm_factor(comparison):
     is divided by 2 to get control variance.
```

Comment:
Is there any way this can be explained a little more clearly? And the Sefarian should go in "References" or whatever convention we've been using at the end of docstrings. Matt Newman at NOAA ESRL (very active in predictability community) was asking what "fac" meant on our docs page and I didn't know. See comments later on clarifying that in docstrings...

Reply:
Actually it wasn't modified here. Can you add to the normalized metrics docstrings a line for `fac`? See e.g. NMSE here: https://climpred.readthedocs.io/en/stable/metrics.html#normalized-mean-square-error-nmse. There's a vague "fac" in the equation with no explanation to a user what that is.

Either that should be taken out since it's done internally or should be explained that it's 2 for certain perfect model comparisons.

Reply:
I now explain it more in detail.